src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/DefaultComesLastCheck.java

```diff
@@ -66,22 +76,74 @@
         return getAcceptableTokens();
     }
 
+    /**
+     * Whether to allow default keyword not in last but surrounded with case.
+     * @param newValue whether to ignore checking.
+     */
+    public void setskipIfLastAndSharedWithCase(boolean newValue) {
```

Comment:
`skip` should be capital `S`. I'm surprised it still recognizes this as a setter.